src/cmd-oscontainer

```diff
@@ -52,6 +52,9 @@ def oscontainer_extract(containers_storage, src, dest,
         run_verbose(["ostree", "--repo="+dest, "pull-local", src_repo, commit])
     finally:
         subprocess.call(['podman', rootarg, 'umount', cid])
+    if args.ref is not None:
+        run_verbose(["ostree", "--repo="+dest, "refs", '--create='+args.ref, commit])
```

Comment:
why not just `pull-local` by ref above instead of commit ? Wouldn't that create the ref?

Reply:
or will there never be a ref inside an oscontainer repo?

Reply:
I don't want there to be a ref inside an oscontainer repo since it adds confusion.